src/js/modules/ValidationError.js

```diff
@@ -0,0 +1,21 @@
+class ValidationError extends Error {
+  constructor(msg, errorEvent) {
+    super(msg);
+    this.errorEvent = errorEvent;
+  }
+}
+
+class NameIsFalsyError extends ValidationError {
+  constructor(errorEvent) {
+    super('이름을 입력해주세요.', errorEvent);
+  }
+}
```

Comment:
```suggestion
class NameIsFalsyError extends ValidationError {
  static MESSAGE = '이름을 입력해주세요.';
  constructor(errorEvent) {
    super(NameIsFalsyError.MESSAGE, errorEvent);
  }
}
```

이렇게 class의 상수로 정의해서 사용할 수도 있답니다!